ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -72,6 +72,29 @@
                     window.alert("Issue while saving draft");
                 }
             }, "saveError");
+            this.#setupAutoSave(self.getModel());
+        }
+
+        /**
+         * Register time based auto save
+         * @param formModel.
+         */
+         #setupAutoSave(formModel) {
+            const autoSaveProperties = formModel?.properties?.['fd:autoSave'];
+            const enableAutoSave = autoSaveProperties?.['fd:enableAutoSave'];
+            if (enableAutoSave) {
+                const autoSaveStrategyType = autoSaveProperties['fd:autoSaveStrategyType'];
+                const autoSaveInterval = autoSaveProperties['fd:autoSaveInterval'];
+                const saveEndPoint = FormView.Utils.getContextPath() + '/adobe/forms/af/save/' + formModel.id;
+                if (autoSaveStrategyType === 'time' && autoSaveInterval) {
+                    console.log("Registering time based auto save");
```

Comment:
Is there alerting enabled on save API? 
I'm also expecting many errors due to missing USC configs that must be reported as 400 rather than 5××.